.idea/.name

```diff
@@ -0,0 +1 @@
+Score
```

Comment:
`.idea/` files shouldn't be tracked; they should be included in the `.gitignore`. Here's a good template for that:

```
# Gradle files
.gradle/
build/

# Local configuration file (sdk path, etc)
local.properties

# Log/OS Files
*.log

# Android Studio generated files and folders
captures/
.externalNativeBuild/
.cxx/
*.apk
output.json

# IntelliJ
*.iml
.idea/
misc.xml
deploymentTargetDropDown.xml
render.experimental.xml

# Keystore files
*.jks
*.keystore

# Google Services (e.g. APIs or Firebase)
app/google-services.json

# Android Profiling
*.hprof

# Secrets
secrets.properties
```